pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -806,7 +807,7 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, podStatus *kubecontaine
 			// being created. If the pod has been deleted then it's
 			// not a real error.
 			// TODO: this is probably not needed now that termination is part of the sync loop
-			if m.podStateProvider.ShouldPodContentBeRemoved(pod.UID) {
+			if m.podStateProvider.IsPodTerminationRequested(pod.UID) {
```

Comment:
please explain (in commit msg) why we change this. It's not obvious at all.

Reply:
This was already changed to appropriate upstream PR https://github.com/kubernetes/kubernetes/pull/104268